mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx

```diff
@@ -76,6 +77,17 @@ type AITransformationTaskpaneState = {
 }
 
 const NUMBER_OF_ATTEMPTS_TO_GET_COMPLETION = 3;
+const LOADING_HINTS = [
+    'Click the Graph button in the toolbar to generate graphs.',
+    'You can ask Mito AI to transform any sheets in your mitosheet.',
+    'Mito can merge multiple dataframes together.',
+    'Mito AI can create pivot tables.',
```

Comment:
Hmm I think this might actually be more misleading than helpful. People are going to think that they can generate Mito Pivot tables using AI.